api/pkg/apis/v1alpha1/providers/stage/materialize/materialize.go

```diff
@@ -161,7 +161,7 @@ func (i *MaterializeStageProvider) Process(ctx context.Context, mgrContext conte
 					catalog.Spec.Name = name
 					catalog.ObjectMeta.Name = name
 					catalog.Spec.SiteId = i.Context.SiteInfo.SiteId
-					objectData, _ := json.Marshal(catalog.Spec)
+					objectData, _ := json.Marshal(catalog)
```

Comment:
Actually I think this should be catalog.Spec - this is the case when a catalog is wrapped in another catalog. We are materializing the embedded catalog, not the catalog itself

Reply:
Changed to catalog.Spec.Property as discussed.